controllers/controllers/networking/integration/suite_integration_test.go

```diff
@@ -33,6 +34,7 @@ func TestNetworkingControllers(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
+	SetDefaultEventuallyTimeout(10 * time.Second)
```

Comment:
Do all of these assertions have the potential to take this long, or is it just a subset?

I guess some of them depend on changes made by the Services reconciler and Contour's HTTPProxy controller which is why it's so variable?

cc @acosta11 and @davewalter

Reply:
Well, all the eventuallies in the test had a two second timeout so I guess they take some time. From my point of view, we should not bother with how long the eventually timeout is as long as there are no performance requirements. I personally prefer a test to always pass within 10 seconds, rather than occasionally fail due to insufficient eventually timeout.